### PR TITLE
OHRI-2053 CaCx - update metadata to use config instead of constants

### PR DESCRIPTION
**Pull request by arodidev**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary

Refactor esm-cervical-cancer-app to use values from config instead of constants

### CODE REVIEW

**Review by ODORA0 — CHANGES_REQUESTED** [review]
@arodidev Let us also add configuration for the form names, just add a new node formNames and do the same for all references of the forms in the components. See example [here](https://github.com/UCSF-IGHS/openmrs-esm-ohri/pull/1735/files#diff-4971ea2a41aa5251ce5565f4ba32d5c44eda061777f5e4b3471686a51063fed1R85)

File: packages/esm-cervical-cancer-app/src/constants.ts [hunk 1]
```diff
@@ -1,15 +0,0 @@
-//cacx registration
```
Comment (pirupius) [inline packages/esm-cervical-cancer-app/src/constants.ts:1]: Please delete this file if it's empty

File: packages/esm-cervical-cancer-app/src/config-schema.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      screeningAndCancerTreatmentForm: 'Screening and Cancer Treatment Form',
+      cervicalCancerRegistrationForm: 'Cervical Cancer Registration Form',
+    },
+  },
+};
+
+export interface ConfigObject {
+  identifiers: Object;
+  encounterTypes: Object;
+  obsConcepts: Object;
```
Comment (pirupius) [inline packages/esm-cervical-cancer-app/src/config-schema.ts:41]: add `formNames`

**Review by pirupius — CHANGES_REQUESTED** [review]
Some final tweaks before we can get this in

**Review by pirupius — APPROVED** [review]

**Review by ODORA0 — APPROVED** [review]

**Pull request merged**